modules/codegen/src/smithy4s/codegen/SmithyBuildJson.scala

```diff
@@ -25,6 +25,10 @@ import smithy4s.codegen.internals.SmithyBuildMavenRepository
 import scala.collection.immutable.ListSet
 
 private[codegen] object SmithyBuildJson {
+
+  val protocolDependency =
+    s"com.disneystreaming.smithy4s:smithy4s-protocol:${BuildInfo.version}"
```

Comment:
**nitpick:** wouldn't hurt us to add `organization` and `protocolArtifactName` as buildinfo keys. Totally fine if you don't, though.

Reply:
Were smithy4s to need to support a build tool with a slightly different format (bleep ?) then it'd save copying and hacking a magic string.

Reply:
Feel free to add it in this PR, if you'd like